src/components/CommonLinks.tsx

```diff
@@ -83,14 +83,10 @@ function UploadANewFile({
 }
 
 const RegulationBSectionUrls = {
-  '§ 1002.109(a)(1)(ii)':
-    '/2023/05/31/2023-07230/small-business-lending-under-the-equal-credit-opportunity-act-regulation-b#p-4302',
-  '§ 1002.109(b)(10)':
-    '/2023/05/31/2023-07230/small-business-lending-under-the-equal-credit-opportunity-act-regulation-b#p-4322',
-  '§ 1002.109(b)(3)':
-    '/2023/05/31/2023-07230/small-business-lending-under-the-equal-credit-opportunity-act-regulation-b#p-4309',
-  '§ 1002.109(b)(9)':
-    '/2023/05/31/2023-07230/small-business-lending-under-the-equal-credit-opportunity-act-regulation-b#p-4733',
+  '§ 1002.109(a)(1)(ii)': '/1002/109/#a-1-ii',
+  '§ 1002.109(b)(10)': '/1002/109/#b-10',
```

Comment:
Keeping this `/1002/109/` base to keep things really flexible over all the iRegs. 👍